src/components/DateTimePicker/index.js

```diff
@@ -51,11 +53,17 @@ function PickerComponent(props, ref) {
         },
     }));
 
+    const context = useContext(AppContext);
+    const contextLocale = (context && context.locale) || 'en-US';
```

Comment:
this is not needed since from context the 'en-US' is the default value then you can:
```const locale = localeProp || context.locale;```

Reply:
what if i don't use application component?...